### PR TITLE
Update CHANGELOG.md and version in INSTALL.md

### PR DESCRIPTION
**Pull request by GodloveD**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,7 +9,55 @@ _With the release of `v3.0.0`, we're introducing a new changelog format in an at
...
+  - Introduced a new basic framework for creating and managing plugins
+  - Added the ability to create containers through multi-stage builds
+  - Created the concept of a "remote" SCS endpoint and added the ability for users and admins to set through conf files 
+  - Streamlined the key container signing, verifying, and key management workflows
+  - Added caching for images from Singularity Hub
+  - Made it possible to compile Singularity outside of `$GOPATH`
+  - Added a json partition to SIF files for OCI configuration when building from an OCI source
+  - Full integration with Singularity desktop for MacOS code base
+  - Added --nocolor flag to Singularity client to disable color in logging
+  - Scientific Filesystem app, when doesn't exist, exits with clear message.
```
Comment (bauerm97) [inline CHANGELOG.md]: This probably shouldn't be included in changelog.

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,7 +9,55 @@ _With the release of `v3.0.0`, we're introducing a new changelog format in an at
...
+
+# v3.2.0 - [2019.04.11]
+
+## New features / functionalities
+  - Introduced a new basic framework for creating and managing plugins
+  - Added the ability to create containers through multi-stage builds
+  - Created the concept of a "remote" SCS endpoint and added the ability for users and admins to set through conf files 
+  - Streamlined the key container signing, verifying, and key management workflows
+  - Added caching for images from Singularity Hub
+  - Made it possible to compile Singularity outside of `$GOPATH`
```
Comment (bauerm97) [inline CHANGELOG.md:21]: Also probably doesn't need to be included here

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,7 +9,55 @@ _With the release of `v3.0.0`, we're introducing a new changelog format in an at
...
-# Changes Since v3.1.1
+# Changes Since v3.2.0
+
+# v3.2.0 - [2019.04.11]
+
+## New features / functionalities
+  - Introduced a new basic framework for creating and managing plugins
+  - Added the ability to create containers through multi-stage builds
+  - Created the concept of a "remote" SCS endpoint and added the ability for users and admins to set through conf files 
+  - Streamlined the key container signing, verifying, and key management workflows
```
Comment (bauerm97) [inline CHANGELOG.md]: Also probably not necessary. It's pretty vague and isn't a new feature or function.

File: CHANGELOG.md [hunk 4]
```diff
@@ -9,7 +9,53 @@ _With the release of `v3.0.0`, we're introducing a new changelog format in an at
...
 
-# Changes Since v3.1.1
+# Changes Since v3.2.0
+
+# v3.2.0 - [2019.04.11]
+
+## New features / functionalities
+  - Introduced a new basic framework for creating and managing plugins
+  - Added the ability to create containers through multi-stage builds
+  - Created the concept of a "remote" SCS endpoint and added the ability for users and admins to set through conf files
```
Comment (ikaneshiro) [inline CHANGELOG.md]: `ability to set _them_ through _CLI_ and conf files`

**Review by ikaneshiro — APPROVED** [review]

**Pull request merged**